app/core/Tracker.coffee

```diff
@@ -63,7 +61,9 @@ module.exports = class Tracker extends CocoClass
 
     if me.isTeacher(true) and @segmentLoaded and not me.get('unsubscribedFromMarketingEmails')
       traits.createdAt = me.get 'dateCreated'  # Intercom, at least, wants this
-      analytics.identify me.id, traits
+
+      # TODO remove intercom config once it's disabled in segment
```

Comment:
Do we have an asana task/reminder for these? I would personally forget otherwise, haha.

Reply:
+1.

Reply:
https://app.asana.com/0/1161281339824033/1166295932229329/f done!